cpanspec

```diff
@@ -190,7 +190,7 @@ use warnings;
 use 5.010;
 
 our $NAME    = "cpanspec";
-our $VERSION = '1.84.00';
+our $VERSION = '1.84.01';
```

Comment:
Please don't change the version, that's done by the maintainer directly before the release.